src/main/java/dbimitation/Storage.java

```diff
@@ -0,0 +1,13 @@
+package dbimitation;
+
+import fruitsassortment.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<Fruit, Integer> FRUIT_MAP = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<Fruit, Integer> fruits = new HashMap<>();
```
it's not a constant unless you use Immutable map, so we use regular naming here